src/query/phrase_query/exact_phrase_scorer.rs

```diff
@@ -0,0 +1,291 @@
+use std::cmp::Ordering;
```

Comment:
This file is non slop code that was in we still have refs to slop here but they should be gone :)